app/views/rubygems/_rubygem.html.erb

```diff
@@ -2,7 +2,12 @@
   <span class="gems__gem__info">
     <h2 class="gems__gem__name">
       <%= rubygem.name %>
-      <span class="gems__gem__version"><%= latest_version_number(rubygem) %></span>
+      <span class="gems__gem__version">
+        <%= latest_version_number(rubygem) %>
+      </span>
+      <span class="gems__gem__latest__release__date">
+        <%= version_date_tag(rubygem.latest_version) %>
+      </span>
```

Comment:
Is this an acceptable appearance? 
![Screenshot 2024-02-21 at 21 40 49](https://github.com/rubygems/rubygems.org/assets/14355495/5ce7232e-ff32-45eb-925e-144d96ad067c)